02-oop-inheritance/js/index.js

```diff
@@ -0,0 +1,140 @@
+class EventEmitter {
+  constructor() {
+    this.events = [];
+  }
+
+  on(event, listener) {
+    let eventFound = this.events.find(function(element) {
+      if (event === element.name) return element;
```

Comment:
This is okey, but for reading and maintainability purpose the if block should be 

```
if (condition) {
  statements
}
```  

event if the statement is one line